tests/end_to_end/target_snowflake/__init__.py

```diff
@@ -0,0 +1,54 @@
+import os
+import shutil
+import unittest
+from pathlib import Path
+
+from tests.end_to_end.helpers import assertions, tasks
+from tests.end_to_end.helpers.env import E2EEnv
+
+TEST_PROJECTS_DIR_PATH = "tests/end_to_end/test-project"
+USER_HOME = os.path.expanduser("~")
+CONFIG_DIR = os.path.join(USER_HOME, ".pipelinewise")
+
+
+class TargetSnowflake(unittest.TestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.e2e_env = self.get_e2e_env()
+        self.check_snowflake_credentials_provided()
+        self.check_validate_taps()
+        self.check_import_config()
+        return None
+
+    def tearDown(self) -> None:
+        return super().tearDown()
```

Comment:
same question as `setUp`

Reply:
Not "needed". I just have a habit of writing return statements explicitly. Let me know if you would like it to be removed.